pages/windows/getmac.md

```diff
@@ -0,0 +1,27 @@
+# getmac
+
+> Display the MAC addresses for network controllers on a system.
```

Comment:
I wonder if "Display the MAC addresses of a system" sounds simpler. The network controllers part might be a bit redundant.

Reply:
Makes sense 👍 Keep it simple!